### PR TITLE
chore: update Tabs to be synced

### LINKED ISSUE #104: Synced Tabs
Pages with tabs should use Synced Tabs. 

https://astro-tips.dev/recipes/dynamic-footer-date-for-static-websites/

### PR DESCRIPTION
**Pull request by hkbertoson**
Added Synced Tabs. This closes #104

### CODE REVIEW

File: src/content/docs/recipes/dark-mode.mdx [hunk 1]
```diff
@@ -243,7 +243,7 @@ If you want to use this `window.theme` API inside a normal `<script>`, you might
 If you're using ESLint, there's a good chance you'll run into `'theme' is not defined` due to the [no-undef](https://eslint.org/docs/latest/rules/no-undef) rule. We can add `theme` as a global in `eslintrc.cjs` to solve this.
 
 
-<Tabs>
+<Tabs syncKey="darkMode">
```
Comment (at-the-vr) [inline src/content/docs/recipes/dark-mode.mdx:246]: I don't think synced tabs are required for this tip. Clearly this 2 tabs are entirely different

File: src/content/docs/tips/how-to-add-gsap.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can easily add GSAP to your Astro project!
 <Steps>
 1. Start by installing the `gsap` package using your favorite package manager:
 
-    <Tabs>
+    <Tabs syncKey="gsap">
```
Comment (at-the-vr) [inline src/content/docs/tips/how-to-add-gsap.mdx:13]: Yeah not required in this tip as well

**Review by at-the-vr — COMMENTED** [review]
gave a quick look around the deployment, sharing couple of suggestions. Overall changes are looking Good 🔥

File: src/content/docs/recipes/dark-mode.mdx (hunk 1 above)
Comment (hkbertoson) [reply src/content/docs/recipes/dark-mode.mdx:246]: Got it!

File: src/content/docs/tips/how-to-add-gsap.mdx (hunk 2 above)
Comment (hkbertoson) [reply src/content/docs/tips/how-to-add-gsap.mdx:13]: Sounds great.

File: src/content/docs/recipes/dark-mode.mdx [hunk 3]
```diff
@@ -306,7 +306,7 @@ import { ViewTransitions } from "astro:transitions";
 
 ## Full code
 
-<Tabs>
+<Tabs syncKey="darkMode">
```
Comment (alexanderniebuhr) [inline src/content/docs/recipes/dark-mode.mdx:309]:
```suggestion
<Tabs>
```

**Pull request merged**